src/twisted/conch/test/test_window.py

```diff
@@ -66,3 +67,49 @@ def test_parent(self) -> None:
         scrolled = ScrolledArea(widget)
         self.assertIs(widget.parent, scrolled._viewport)
         self.assertIs(scrolled._viewport.parent, scrolled)
+
+
+class SelectionTests(TestCase):
+    """
+    Tests for L{Selection}, a widget which allows to select item from
+    list of items.
+    """
+
+    seq = [f"{x}".encode("ascii") for x in range(10)]
```

Comment:
Can this be moved inside `test_selection` ?

Do they need to be class members ?